fedora-messaging.spec

```diff
@@ -0,0 +1,132 @@
+%{!?__python2: %global __python2 %__python}
+%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+
+%global pkgname fedora-messaging
+%global srcname fedora_messaging
+%global desc \
+Tools and APIs to make working with AMQP in Fedora easier.
+
+
+Name:           %{pkgname}
+Version:        1.0.0
+Release:        0.1.a1%{?dist}
+Summary:        Set of tools for using Fedora's messaging infrastructure
+
+License:        GPLv2+
+URL:            https://github.com/fedora-infra/fedora-messaging
+Source0:        https://github.com/fedora-infra/%{name}/archive/v%{version}a1/%{srcname}-%{version}a1.tar.gz
+BuildArch:      noarch
+
+BuildRequires:  python2-devel
+BuildRequires:  python2-blinker
+BuildRequires:  python2-click
+BuildRequires:  python2-jsonschema
+BuildRequires:  python2-pytoml
+BuildRequires:  python2-pika
+BuildRequires:  python2-six
+BuildRequires:  python2-pytest
+BuildRequires:  python2-twisted
+BuildRequires:  python2-pyOpenSSL
+#BuildRequires:  python2-pytest-twisted
```

Comment:
It this not packaged or do we not need it?

Reply:
It's not packaged and we only need it for the Twisted unit tests. So I disabled those tests.